plugins/modules/template_intent.py

```diff
@@ -2479,7 +2494,8 @@ def handle_import(self, config):
                 # "payload": "{0}".format(payload)
                 "payload": payload
             }
-            self.log(str(_import_project))
+            self.log("Import Project Details from the Playbook: {0}"
```

Comment:
self.log("Importing project details from the playbook: {0}".format(_import_project), "DEBUG")